extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -434,6 +432,32 @@ public void startSearch(List<String> serialNumbers) {
             }
         }
     }
+    
+    /**
+     * Iterate through lightStatusListeners and notify them about a changed ot added light state.
+     * @param fullLight
```

Comment:
> Wow, what hard reviews!

Sorry about that. But I think we have to look for a consistent and correct formatted code and documentation.
We could also not complain such stuff, but I don't think this will improve the framework.